Doppler.PushContact/Models/Domain.cs

```diff
@@ -0,0 +1,9 @@
+namespace Doppler.PushContact.Models
+{
+    public class Domain
+    {
+        public string Name { get; set; }
+
+        public DomainStatus Status { get; set; }
```

Comment:
Is this `DomainStatus` enum necessary? Maybe we can use bool instead of, something like this:

```suggestion
        public bool IsPushFeatureEnabled { get; set; }
```

Reply:
in this case, this property should only have one function, enable or disable push, if we need another functionality we should create another property for that. I think it's okay to leave this property as bool and rename it to IsPushFeatureEnabled.

Reply:
I like @hrocca's proposal.